pkg/manager/router/backend_observer.go

```diff
@@ -160,6 +162,7 @@ func (bo *BackendObserver) observe(ctx context.Context) {
 			select {
 			case <-time.After(wait):
 			case <-bo.refreshChan:
+				refresh = true
```

Comment:
Gateway may call `Router.RefreshBackend` to refresh. I don't remember when Gateway may call it. Please ensure that in that case, adding one backend back is expected.
Besides, the word `refresh` is vague. Every time I'll think for a while what the parameter means.

Reply:
I have stopped Gateway to call RefreshBackend in my PR and it's works well. The router call GetBackendList every 3 seconds. It's fast enough for Gateway.
How about "force"? it sounds more natural in serverlerss scenario. But I am not sure if it make sense in TiProxy.

Reply:
No, just add a comment on the function definition.